universal-application-tool-0.0.1/app/views/applicant/ApplicantProgramSummaryView.java

```diff
@@ -71,6 +77,23 @@ public Content render(
     // Add submit action (POST).
     String submitLink =
         routes.ApplicantProgramReviewController.submit(applicantId, programId).url();
+
+    Tag continueOrSubmitButton;
+    if (completedBlockCount == totalBlockCount) {
+      continueOrSubmitButton =
+          submitButton(messages.at(MessageKey.BUTTON_SUBMIT.getKeyName()))
+              .withClasses(
+                  ReferenceClasses.SUBMIT_BUTTON, ApplicantStyles.BUTTON_SUBMIT_APPLICATION);
+    } else {
+      String applyUrl = routes.ApplicantProgramsController.edit(applicantId, programId).url();
+      continueOrSubmitButton =
+          a().attr(HREF, applyUrl)
```

Comment:
praise: Anchor tag instead of a button here is good for having proper roles in our HTML and for screen readers, etc. 👍